### PR TITLE
Bundler fixes

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: analyzer/src/funTest/kotlin/integration/AbstractIntegrationSpec.kt [hunk 1]
```diff
@@ -76,7 +76,9 @@ abstract class AbstractIntegrationSpec : StringSpec() {
     protected lateinit var downloadResult: Downloader.DownloadResult
 
     override fun beforeSpec(spec: Spec) {
-        outputDir = createTempDir(ORT_NAME, javaClass.simpleName)
+        // Do not use the usual simple class name as the suffix here to shorten the path which otherwise gets too long
```
Comment (fviernau) [inline analyzer/src/funTest/kotlin/integration/AbstractIntegrationSpec.kt:79]: How many characters did the outputDir (as path string) actually have when it exceeded 256?
e.g. did it already exceed or did the exceed happen inside of `Download().download()`?

Comment (sschuberth) [reply analyzer/src/funTest/kotlin/integration/AbstractIntegrationSpec.kt:79]: It was something in the 26x range, I do not remember exactly, but removing the "SimpleFormIntegrationTest" suffix from the path really made the difference already here.

Comment (fviernau) [reply analyzer/src/funTest/kotlin/integration/AbstractIntegrationSpec.kt:79]: Thanks!

**Pull request merged**